ApplicationLibCode/Commands/OsduImportCommands/RicWellPathsImportOsduFeature.cpp

```diff
@@ -0,0 +1,133 @@
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  ResInsight is distributed in the hope that it will be useful, but WITHOUT ANY
+//  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+//  FITNESS FOR A PARTICULAR PURPOSE.
+//
+//  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+//  for more details.
+//
+/////////////////////////////////////////////////////////////////////////////////
+
+#include "RicWellPathsImportOsduFeature.h"
+
+#include "RiaApplication.h"
+#include "RiaGuiApplication.h"
+#include "RiaLogging.h"
+#include "RiaOsduConnector.h"
+#include "RiaPreferences.h"
+
+#include "RigWellPath.h"
+
+#include "RiaPreferencesOsdu.h"
+#include "RimFileWellPath.h"
+#include "RimOilField.h"
+#include "RimOsduWellPath.h"
+#include "RimProject.h"
+#include "RimTools.h"
+#include "RimWellPathCollection.h"
+#include "RimWellPathImport.h"
+
+#include "RiuMainWindow.h"
+#include "RiuWellImportWizard.h"
+
+#include "cvfObject.h"
+
+#include <QAction>
+#include <QDir>
+
+CAF_CMD_SOURCE_INIT( RicWellPathsImportOsduFeature, "RicWellPathsImportOsduFeature" );
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RicWellPathsImportOsduFeature::onActionTriggered( bool isChecked )
+{
+    RiaApplication* app = RiaApplication::instance();
+    if ( !app->project() ) return;
+
+    // Update the UTM bounding box from the reservoir
+    app->project()->computeUtmAreaOfInterest();
+
+    QString wellPathsFolderPath = QStandardPaths::writableLocation( QStandardPaths::CacheLocation ) + QString( "/wellpaths/" );
+    QDir::root().mkpath( wellPathsFolderPath );
+
+    if ( !app->project()->wellPathImport() ) return;
+
+    // Keep a copy of the import settings, and restore if cancel is pressed in the import wizard
+    QString copyOfOriginalObject = app->project()->wellPathImport()->writeObjectToXmlString();
+
+    if ( !app->preferences() ) return;
+
+    RimProject* project = RimProject::current();
+    if ( !project ) return;
+
+    if ( project->oilFields.empty() ) return;
+
+    RimOilField* oilField = project->activeOilField();
+    if ( !oilField ) return;
+
+    RiaPreferencesOsdu* osduPreferences = app->preferences()->osduPreferences();
+
+    const QString server         = osduPreferences->server();
+    const QString dataParitionId = osduPreferences->dataPartitionId();
+    const QString authority      = osduPreferences->authority();
+    const QString scopes         = osduPreferences->scopes();
+    const QString clientId       = osduPreferences->clientId();
+
+    auto m_osduConnector = new RiaOsduConnector( RiuMainWindow::instance(), server, dataParitionId, authority, scopes, clientId );
```

Comment:
Remove m_ if local variable and put into a unique pointer

Reply:
Fixed.